spec/index.html

```diff
@@ -10457,6 +10457,12 @@ <h3>Grammar</h3>
           <li>The expression argument of an aggregate function can not contain an aggregate function.</li>
           <li>Only custom aggregate functions use the <code>DISTINCT</code> keyword
             in a <a href="#rFunctionCall">function call</a>.</li>
+          <li>A <a href="#rReifier">reifier</a> or
+            <a href="#rAnnotationBlockPath">annotation syntax</a>
+            is only permitted after a triple when the property position is
+            a simple path (IRI, the keyword <code>a</code> or a variable),
```

Comment:
The terminal comma prevents potential reader confusion.
```suggestion
            a simple path (an IRI, the keyword <code>a</code>, or a variable),
```